src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,58 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String email;
```

Comment:
better to add annotation @Column(unique = true)  above field email

Reply:
i think this annotation work in DB. But i use check optionalUserByEmail.isPresent() in register method in AuthenticationServiceImpl

Reply:
let's see what the mentor will say